SUPPORT.md

```diff
@@ -0,0 +1,8 @@
+# How to get support from the Libero Community
+The best place to start for support on any aspect of Libero is the [Libero Commmunity Slack workspace](https://libero.pub/join-slack/). Invitiations are open so just join the conversation and ask your question in the relevant channel.
+
+The community has members from a number of organisations and the project lead, eLife, have their [teams listed in the Libero GitHub organisation](https://github.com/orgs/libero/teams/elife-sciences/teams). If you want to mention someone on Slack, in an issue or in a pull request, then you can find out who we are by using [the teams page](https://github.com/orgs/libero/teams/elife-sciences/teams).
```

Comment:
https://github.com/orgs/libero/teams/elife-sciences/teams is not a publicly accessible URL

Reply:
(need to be in the Libero organization to see it, at least)

Reply:
I've updated to the people page which is less useful (no team detail) but it will need people to make their membership of `Libero` `visible`